install_config/topics/ocp_osp_sdn.adoc

```diff
@@ -36,3 +36,9 @@ To enable Kuryr SDN, your environment must meet the following requirements:
 * Neutron Trunk ports extension enabled
 * If ML2/OVS Neutron driver is used the OpenvSwitch firewall driver must be
 used, instead of the ovs-hybrid one.
+
+[IMPORTANT]
+====
+To use Kuryr with {rh-openstack-first} 13.0.13, the Kuryr container images must
```

Comment:
```suggestion
To use Kuryr with OpenStack 13.0.13, the Kuryr container images must
```
The 3.11 docs apparently don't use the OSP common attributes, yet. Using the old style for now.